pyomo/contrib/doe/fim_doe_tutorial.ipynb

```diff
@@ -0,0 +1,2994 @@
+      "\n",
+      "Number of objective function evaluations             = 5\n",
+      "Number of objective gradient evaluations             = 5\n",
+      "Number of equality constraint evaluations            = 5\n",
+      "Number of inequality constraint evaluations          = 0\n",
+      "Number of equality constraint Jacobian evaluations   = 5\n",
+      "Number of inequality constraint Jacobian evaluations = 0\n",
+      "Number of Lagrangian Hessian evaluations             = 4\n",
+      "Total CPU secs in IPOPT (w/o function evaluations)   =      0.009\n",
+      "Total CPU secs in NLP function evaluations           =      0.001\n",
+      "\n",
+      "EXIT: Optimal Solution Found.\n",
+      "W[K_AUG]...\t[K_AUG_ASL]No n_rhs declared.\n",
+      "W[K_AUG]...\t[K_AUG_ASL]Target log10mu:= -11.\n",
+      "W[K_AUG]...\t[K_AUG_ASL]This problem has no degrees of freedom\n",
+      "Pass the option square_override for the desired behaviour\n",
+      "W[K_AUG_ASL]...\t[K_AUG_ASL]No ipopt_zL_in suffix declared, setting zL = 0.\n",
+      "W[K_AUG_ASL]...\t[K_AUG_ASL]No ipopt_zU_in suffix declared, setting zU = 0.\n",
+      "W[K_AUG]...\t[K_AUG_ASL]No f_timestamp suffix declared, Fallback to default writing mode.\n",
+      "I[K_AUG]...\t[K_AUG_ASL] Filename for dot_sens dsdp_in_.in\n",
+      "W[K_AUG]...\t[K_AUG_ASL]dsdp for linear C(x) + I*p = 0 override.\n",
+      "E[K_AUG]...\t[SIGMA_COMPUTE]is x[88] = xlb?\n",
+      "E[K_AUG]...\t[SIGMA_COMPUTE]is x[88] = xlb?\n",
+      "E[K_AUG]...\t[SIGMA_COMPUTE]is x[89] = xlb?\n",
+      "E[K_AUG]...\t[SIGMA_COMPUTE]is x[89] = xlb?\n",
+      "E[K_AUG]...\t[SIGMA_COMPUTE]is x[90] = xlb?\n",
+      "E[K_AUG]...\t[SIGMA_COMPUTE]is x[90] = xlb?\n",
+      "E[K_AUG]...\t[SIGMA_COMPUTE]is x[91] = xlb?\n",
+      "E[K_AUG]...\t[SIGMA_COMPUTE]is x[91] = xlb?\n",
+      "E[K_AUG]...\t[SIGMA_COMPUTE]Unresolved sigma values, is the mu computation successful? or is this a square problem?\n",
+      "Check the sigma_out file with the PRINT_VERBOSE (cmake) option set to 0\n",
+      "[K_AUG] 0.1.0, Part of the IDAES PSE framework\n",
+      "Please visit https://idaes.org/\n",
+      ": dsdp_mode=\n",
+      "dsdp_mode=\n",
+      "W[K_AUG]...\t[MA57_DRIVER]\n",
+      "\n",
+      "\t\tCould not fix the accuracy of the problem.\n",
+      "\t\tTry re-writing the problem or give a different point or change \"max_refinement_steps\"\n",
+      "\t\tWarning: results might be incorrect.\n",
+      "\t\tCurrent residual ratio 0.0424279; Max residual ratio 1e-10.\n",
+      "\n",
+      "I[K_AUG]...\t[K_AUG_ASL]File read successfully.\n",
+      "I[K_AUG]...\t[K_AUG_ASL]Number of Right hand sides: 0\n",
+      "I[K_AUG]...\t[K_AUG_ASL]Number of variables       : 865\n",
+      "I[K_AUG]...\t[K_AUG_ASL]Number of constraints     : 865\n",
+      "I[K_AUG]...\t[K_AUG_ASL]Number of valid n_dof     : 0\n",
+      "I[K_AUG]...\t[ADJUST_MU]\tWarning strict complementarity (lb) is not within tol for var:88\n",
```

Comment:
This warning message is also a concern. There may be redundant bounds in the problem or another issue with the model.